community/modules/compute/schedmd-slurm-gcp-v6-nodeset/main.tf

```diff
@@ -35,6 +35,11 @@ locals {
   public_access_config = var.disable_public_ips ? [] : [{ nat_ip = null, network_tier = null }]
   access_config        = length(var.access_config) == 0 ? local.public_access_config : var.access_config
 
+  ghpc_startup_script = [{
+    filename = "ghpc_nodeset_startup.sh"
```

Comment:
Is this problematic that each nodeset has the same filename. From your testing it seems not.

Reply:
ack